CHANGELOG.md

```diff
@@ -37,6 +37,11 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
+### API Breaking Changes
+
+* (codec) [\#9521](https://github.com/cosmos/cosmos-sdk/pull/9521) Removed deprecated `clientCtx.JSONCodec` from `client.Context`.
+* (codec) [\#9521](https://github.com/cosmos/cosmos-sdk/pull/9521) Rename `EncodingConfig.Marshaler` to `Codec`.
```

Comment:
I totally agree with those 2 changes, but otoh I also hear community feedback saying that there are too many breaking changes between releases.

So I'm not 100% sure of adding this change, imo it doesn't cost much to us to keep them deprecated, and give more time to app devs to adapt.

Reply:
We should do that for consistency. In the past we renamed `Marshaler` to `Codec`, and we forgot about renaming the part in `EncodingConfig`.

Reply:
I see. Then we might as well do it here and now.

Reply:
@alexanderbez  - the related rename is part of the 0.43 release, we forgot to update `EncodingConfig`.